setup.py

```diff
@@ -472,12 +472,12 @@ def readme():
 # in the correct place unless we use 'package_dir' and this trick.
 # 1. Here we list files as they should appear in packages for end-users
 # 2. In 'package_dir' we defer the package name to the local file path
-packages += map(lambda x: f"sisl_toolbox.{x}", find_packages("toolbox"))
+packages += map(lambda x: f"toolbox.{x}", find_packages("toolbox"))
 
 # Also ensure we have all "pxd" files
 package_data = {p: ["*.pxd"] for p in packages}
 # Add toolbox data
-package_data["sisl_toolbox.siesta.minimizer"] = ["basis.yaml", "pseudo.yaml"]
+package_data["toolbox.siesta.minimizer"] = ["basis.yaml", "pseudo.yaml"]
```

Comment:
Don't change here as well.

Reply:
same reason... couldn't find the folders...

Reply:
Same as above please remove from this pr